### DISCUSSION TITLE
add "-skipsettings" option to skip settings windows and directly connect to server and launch player

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#597] [Ideas]
If this option is used but no settings are present there should be an error asking you to configure syncplay before using this option.

**Comment by Et0h** [comment]
I can see why some people might want more control as to when the GUI settings config will appear (but in a way which makes it easy to get back to the settings dialogue if needed).

Before shared playlists the primary way to load Syncplay was to open a file with Syncplay as you had to manually load the file. As such, we could easily make it so people could choose to show GUI settings when they launched directly but allow people to skip it when they opened Syncplay with a file. Because of shared playlists people are no longer expected to load Syncplay via 'open with' and so this means there is now no good way to simply load Syncplay without the GUI Settings.

As part of this legacy we have --force-gui-prompt as an option, so I guess a question would be: If we were to add such an option, should it be called --skip-settings or --skip-gui-prompt or something else.

**Comment by soredake** [comment]
This function is now implement in syncplay-android https://github.com/yuroyami/syncplay-android/issues/62#issuecomment-1574499645
